test/content/support.js

```diff
@@ -241,4 +241,17 @@ function resetDB() {
 	}).then(function() {
 		return Zotero.Schema.schemaUpdatePromise;
 	});
-}
\ No newline at end of file
+}
+
+
+/**
+ * Imports an attachment from a test file.
+ * @param {string} filename - The filename to import (in data directory)
+ * @returns {Promise} as returned by Zotero.Attachments.importFromFile
+ */
+function importFromTestFile(filename) {
```

Comment:
To clarify, could we call this `importFileAttachment` (it's not clear what we're importing from test file)